source/eventHandler.py

```diff
@@ -263,21 +263,23 @@ def isMenuItemOfCurrentFocus(self) -> bool:
 			isinstance(self._obj, IAccessible.IAccessible)
 			and isinstance(lastFocus, IAccessible.IAccessible)
 			and self._obj.IAccessibleRole in (
-				oleacc.ROLE_SYSTEM_MENUITEM, IA2.IA2_ROLE_CHECK_MENU_ITEM, IA2.IA2_ROLE_RADIO_MENU_ITEM)
+				oleacc.ROLE_SYSTEM_MENUITEM,
+				IA2.IA2_ROLE_CHECK_MENU_ITEM,
+				IA2.IA2_ROLE_RADIO_MENU_ITEM
+			)
 			and lastFocus.IAccessibleRole == oleacc.ROLE_SYSTEM_MENUPOPUP
-			and self._obj.parent):
+			and self._obj.parent
+		):
 			return False
 
 		# Check that the old focus is a descendant of the new focus.
 		ancestor = self._obj.parent
 		while ancestor is not None:
 			if ancestor == lastFocus:
-				log.debugWarning(
+				log.debug(

```

Comment:
Is there a reason why the log level changed here?
Debug warning makes sense to me in this case, as it is a warning message.

Reply:
```suggestion
				log.debugWarning(
```